docs/source/unit-testing.md

```diff
@@ -155,8 +144,31 @@ shared libraries, models, etc.
         assert type(result) is ProcessedExampleModel
 ```
 
+### Actors's test dependencies
+
+If your **actor's tests** require a special package for their execution, create a
+Makefile in the [actor's root directory](repository-dir-layout.html) with an
+`install-deps` target calling `yum install -y`.
+
+```sh
+$ cat actors/myactor/Makefile
+install-deps:
+	yum install -y my-tests-need-this-pkg
+```
+
+Note: Dependencies defined the way mentioned above is for test execution only.
+If your actor requires any package when executed as part of a workflow, it
+needs to be specified in a
+[leapp-repository specfile](https://github.com/oamg/leapp-repository/blob/master/packaging/leapp-repository.spec).
+
 ## Running the tests
 
+### Preparing the environment
+
+To execute unit tests of actors from all Leapp repositories in the
+`leapp-repository` GitHub repository, you need to install test dependencies for each
+actor by running  `make install-deps`.
+
 ### Actor's tests
 
 Use _pytest_ to run the tests for your actor. For the test modules to be able to
```

Comment:
I think you can remove the "manual" way using pytest and leave just there the `make test`.